### PR TITLE
initial support for working together with language-server

### PR DESCRIPTION
**Pull request by daimor**
This PR fixes #

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -525,10 +535,50 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
     outputChannel.show(true);
   }
 
+  const languageServerExt = await languageServer();
+  if (!languageServerExt) {
```
Comment (gjsjohnmurray) [inline src/extension.ts:539]: I suggest you add an `outputChannel.appendLine` call in this block to notify user that the language server wasn't found.

**Review by gjsjohnmurray — COMMENTED** [review]
With both extension installed and active, if I disable the LS extension it requires a reload. Things then work as expected, with the ObjectScript extension taking over all the roles. But if I subsequently enable LS no reload is required and we get contributions from both extensions (e.g. double hovers).

Here's an idea for a solution:

- During `activate` gather the Disposables returned from each of the `onXXX` and  `registerXXX` calls that you make only when LS isn't active.
- Use the [`extensions.odDidChange` event](https://code.visualstudio.com/updates/v1_31#_extensions-change-event) to detect when LS gets activated. When this happens, dispose of all the Disposables you gathered during activation.

**Comment by daimor** [comment]
> With both extension installed and active, if I disable the LS extension it requires a reload. Things then work as expected, with the ObjectScript extension taking over all the roles. But if I subsequently enable LS no reload is required and we get contributions from both extensions (e.g. double hovers).
> 
> Here's an idea for a solution:
> 
> * During `activate` gather the Disposables returned from each of the `onXXX` and  `registerXXX` calls that you make only when LS isn't active.
> * Use the [`extensions.odDidChange` event](https://code.visualstudio.com/updates/v1_31#_extensions-change-event) to detect when LS gets activated. When this happens, dispose of all the Disposables you gathered during activation.

Done

**Review by gjsjohnmurray — APPROVED** [review]
LGTM and I will merge it immediately.

**Pull request merged**